### ISSUE TITLE
[Issue] Realign the freezethrower's top sprite

### ISSUE THREAD

**Issue by FistsAkimbo** [NightFright2k19/duke3d_widefix#7]
This is technically an animation fix rather than a sprite fix, but could you please adjust the position of the upper sprite of the freezethrower's firing animation, so it aligns with the knuckles on the lower sprite? I'd love to use all the fixes in this mod with that fix, but if you (understandly) would rather not, what would be the best tool to use to modify this myself?

**Comment by NightFright2k19** [comment]
I would need a screenshot for this. Note that most of these sprite alignments weren't made by me, but the actual sprite artists who added the widescreen extensions. It should be assumed all alignments are correct, especially nobody has ever complained about them in years. It's kinda hard for me to believe something has been overlooked after all.

(Unless I misunderstand this and it's about a visual flaw on the actual sprite, something that would need to be done with an image editor.)

**Comment by FistsAkimbo** [comment]
Sure thing, I attached the pictures below.

To clarify, I'm not referring to your mod's position of the sprites (which appear to drop the top layer of the freezethrower in the same offset position as the original DN3D) but how in both cases, they aren't being drawn correctly based off of position; for all three upper sprites used in the animation, they're all offset to the right by three, and down by seven.

![video screenshot](https://github.com/NightFright2k19/duke3d_widefix/assets/44789472/9bbbbfa9-b086-4c89-bc6c-5eb30918c311)
![freezethrower misaligned 1](https://github.com/NightFright2k19/duke3d_widefix/assets/44789472/2054de9b-8f7d-4234-8a84-2dd867a39cef)
![freezethrower misaligned 2](https://github.com/NightFright2k19/duke3d_widefix/assets/44789472/eb64c356-a8c5-4350-ab54-e472c4514c71)
![freezethrower misaligned 3](https://github.com/NightFright2k19/duke3d_widefix/assets/44789472/4460aed0-35b4-4254-a64e-ea0fd78d93e5)
![freezethrower correct 1](https://github.com/NightFright2k19/duke3d_widefix/assets/44789472/b6bb0584-7056-4864-83ce-44db2e921cdf)
![freezethrower correct 2](https://github.com/NightFright2k19/duke3d_widefix/assets/44789472/6ff019da-8294-427f-b4dc-b318df0050ef)
![freezethrower correct 3](https://github.com/NightFright2k19/duke3d_widefix/assets/44789472/211bf64c-f9e6-4bf2-b988-9ed4700caab9)

**Comment by NightFright2k19** [comment]
I see what you mean. Please test with the attached test file and report back whether alignments are correct now. Since the lower part seems to be properly in place, I have adjusted the top part of the gun (tiles 2551-2553).

In fact, technically all default alignments are correct since they follow the DOS release. It's the replication of a bug in the original game. Which doesn't mean it shouldn't be addressed, but in the end it's hard to tell which kind of correction is better. Possibly adjusting the top part.

Also:
I assume the Caribbean counterpart of the Freezer does not require these adjustments. I have checked it ingame and it looks as intended with its current settings.


[duke3d_widefix-test.zip](https://github.com/NightFright2k19/duke3d_widefix/files/11609685/duke3d_widefix-test.zip)

**Comment by FistsAkimbo** [comment]
Thanks for that test zip, the alignment is right on the money. However; just like 20th Anniversary World Tour (which also fixes the freezethrower offsets,) there's this bizarre small gap seen in the animations while it's firing, where I'm not sure if it's just how these sprites are drawn by the renderer, or if it's somehow an offset error. I lean towards the former because, every time I stopped mid-firing frame by opening the main menu, the sprites are drawn correctly with no gap, when the display of the game is at a rest. I've included a minor edit that extends the bottom cutoff edge of the base sprite so that it may save you half a minute to edit. But of course, if there's some other proper means of fixing this strange gap, that should probably be the way to go.
![2550 inner edge extended](https://github.com/NightFright2k19/duke3d_widefix/assets/44789472/09f644a6-d452-498f-8907-de9727a9a31f)
![duke0018](https://github.com/NightFright2k19/duke3d_widefix/assets/44789472/fd74e45d-1f9a-4f10-a772-b9f959fa5c45)
